package.json

```diff
@@ -53,12 +53,12 @@
     "@commitlint/cli": "^11.0.0",
     "@commitlint/config-conventional": "^11.0.0",
     "@rollup/plugin-node-resolve": "^10.0.0",
-    "@rollup/plugin-typescript": "^6.0.0",
+    "@rollup/plugin-typescript": "^6.1.0",
```

Comment:
Why the need of updating the libraries? Please let's not do that as part of this PR